addons/website/static/src/js/content/website_root.js

```diff
@@ -35,6 +36,13 @@ var WebsiteRoot = publicRootData.PublicRoot.extend(KeyboardNavigationMixin, {
         });
         return this._super(...arguments);
     },
+    /**
+     * @override
+     */
+    willStart: async function () {
+        this.fullscreenIndication = new FullscreenIndication(this);
+        await this.fullscreenIndication.appendTo(document.body);
+    },
```

Comment:
Missing call to _super. If PublicRoot (the parent class) does anything `willStart` or has any xml-dependencies, they will not be loaded if you don't call super. Even if that's not the case right now, this widget shouldn't break if they're added at a later point:

```js
return Promise.all([
  this._super(...arguments),
  this.fullscreenIndication.appendTo(document.body),
]);
```